code/modules/mob/living/carbon/xenomorph/evo_datum.dm

```diff
@@ -57,15 +57,23 @@
 
 /// Some data to update the UI with the current evolution status
 /datum/evolution_panel/ui_data(mob/living/carbon/xenomorph/xeno)
-	. = list()
+	var/list/data = list()
+
+	data["bypass_evolution_checks"] = SSresinshaping.active
```

Comment:
Why isn't this just in can evolve

Reply:
initially I had it there and it didn't work well with the UI.
I kinda want to make it where if this is active there is some UI indicator but I keep fucking up the UI whenever I try to make changes to it.